tests/python/gaia-ui-tests/gaiatest/gaia_graphics_test.py

```diff
@@ -164,6 +182,7 @@ def edge_scroll(self, frame, direction, dist, release=True):
         if release:
             action.release()
         action.perform()
+        time.sleep(1)  # compensate for the time taken for actual action
```

Comment:
yes, there is no check that can tell me the scroll action (and the rendering of screen) has completed.